protocols/kad/src/query/peers/closest/disjoint.rs

```diff
@@ -743,11 +744,12 @@ mod tests {
     }
 
     impl Arbitrary for Graph {
-        fn arbitrary<G: Gen>(g: &mut G) -> Self {
-            let mut peer_ids = random_peers(g.gen_range(K_VALUE.get(), 200), g)
-                .into_iter()
-                .map(|peer_id| (peer_id.clone(), Key::from(peer_id)))
-                .collect::<Vec<_>>();
+        fn arbitrary(g: &mut Gen) -> Self {
+            let mut peer_ids =
+                random_peers(K_VALUE.get() + (usize::arbitrary(g) % (200 - K_VALUE.get())))
```

Comment:
This change makes sense now but I am a bit worried that it will be hard to follow later without the context of this version upgrade.

Given that this only affects test code, what do you think of adding a `quickcheck-ext` crate to the workspace that is set to `publish = false` where we define a utility function (Perhaps with an extension trait?) to regain the functionality of `gen_range`?

Reply:
That would be much appreciated!

Reply:
Done